average_steer_partitions.py

```diff
@@ -21,6 +21,18 @@ def event_to_analog_value(event: ControlEntry):
     val >>= int32(8)
     return -val
 
+def clean_duplicate_events(events: list):
```

Comment:
Please annotate that this also returns list.

Reply:
done